curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6145f685797bd30df9784e8c.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-41
 
 # --description--
 
-Do not forget to give your `form` a submit button.
+Do not forget to give your form a submit button with the text "Send".
```

Comment:
`form` is a code element name. I don't think we should change this?

```suggestion
Do not forget to give your `form` a submit button with the text "Send".
```

Reply:
The word Send shouldn't be translatable as well

```suggestion
Do not forget to give your `form` a submit button with the text `Send`.
```